### DISCUSSION TITLE
Pre-set special custom value

### DISCUSSION THREAD

**Discussion by bovirus** [benzino77/tasmocompiler#212] [Q&A] [answered]
@benzino77

If I want to make a fork from your source and I want to pre-set some value in custom values how what I can modify in your source code?

The file is /Src/component/AppStepper/CustomParametersStep.js?

Thanks.

**Comment by benzino77** [comment, marked as answer]
Rather you should do this by editing `src/components/AppStepper/FeaturesStep/AvailableFeatures.js`

Pick one of the features you want to use and add your defines to `custom` property:
Example:
```javascript
...
{
    name: 'USE_WEBSERVER',
    value: true,
    show: true,
    description: 'stepFeaturesWebInterfaceDesc',
    tooltip: 'stepFeaturesWebInterfaceTooltip',
    custom:
      '#define TEST_BOV01\n#define TEST_BOV02 123456789\n#define TEST_BOV03 "with some text"',
    boards: ['all'],
  },
...
```

**Comment by bovirus** [comment]
@benzino77 

Thanks for the info.

I want to set by default FIRMWRE LITE then I add to the end of  src/components/AppStepper/FeaturesStep/AvailableFeatures.js

..
**{
    name: 'USE_WEBSERVER',
    value: true,
    show: true,
    description: 'stepFeaturesWebInterfaceDesc',
    tooltip: 'stepFeaturesWebInterfaceTooltip',
    custom:
      '#define FIRMWARE_LITE',
    boards: ['all'],
},**
];
export default availableFeatures;

The bold text is the added text.

**Reply by benzino77** [reply]
I'm not sure it is a good idea to put `#define FIRMWARE_LITE`  during custom Tasmota build.
Heave a look here: https://github.com/benzino77/tasmocompiler/issues/110#issuecomment-763776512

You will probably end up with compiling errors.

**Comment by bovirus** [comment]
@benzino77 

Fork created.
Made the changes (pre-set #define FIRMWARE_LITE)
No error during compilation using the fork.

**Reply by benzino77** [reply]
Still - it is not a good idea. It may compile without errors but resulting binary can have not what you've expected.

Using "predefined" binaries scheme with custom compilation is not a good idea.
